pandas/tseries/frequencies.py

```diff
@@ -185,10 +186,29 @@ def get_offset(name: str) -> DateOffset:
     """
     Return DateOffset object associated with rule name.
 
+    .. deprecated:: 1.0.0
+
     Examples
     --------
     get_offset('EOM') --> BMonthEnd(1)
     """
+    warnings.warn(
+        "get_offset is deprecated and will be removed in a future version, "
+        "use to_offset instead",
+        FutureWarning,
+        stacklevel=2,
+    )
+    return _get_offset(name)
```

Comment:
can this call to_offset instead?

Reply:
No, we still need _get_offset internally, just officially telling users to stay away

Reply:
ok, would rename maybe if you can (followon ok)